src/test/java/hudson/scm/SubversionSCMTest.java

```diff
@@ -61,7 +61,6 @@
 import org.tmatesoft.svn.core.wc.SVNCommitClient;
 import org.tmatesoft.svn.core.wc.SVNRevision;
 import org.tmatesoft.svn.core.wc.SVNStatus;
-import org.tmatesoft.svn.core.wc.SVNWCUtil;
```

Comment:
@christ66 But it is an unnecessary import. What do you propose? A dedicated PR for that???